code/game/objects/items/storage/boxes/job_boxes.dm

```diff
@@ -225,11 +225,26 @@
 	illustration = "heart"
 	foldable_result = null
 	mask_type = null
+	var/random_funny_internals = TRUE
+
+/obj/item/storage/box/survival/hug/PopulateContents()
+	..()
+	if(!random_funny_internals)
+		return
+	var/obj/item/internals = locate(internal_type) in src
+	qdel(internals)
```

Comment:
why not just... set internals to null?

Reply:
I tried that and it send errors, with how the previous proc works or something of the kind.
There is another proc that removes internals like this already so I just copied what we have without thinking much of smoother ways to achieve it.

Reply:
Just fix the bug man

Reply:
It would be more a refactor with how boxes work, which I started in, but Ghommie's suggestion worked already so it is just me being dumb and not a limitation of the old system.